library/source/datadogSdk.bs

```diff
@@ -71,7 +71,7 @@ sub initialize(configuration as object, global as object)
         global.addFields({ datadogLogsAgent: CreateObject("roSGNode", "LogsAgent") })
         global.datadogLogsAgent.site = configuration.site
         global.datadogLogsAgent.clientToken = configuration.clientToken
-        global.datadogLogsAgent.service = configuration.service
+        global.datadogLogsAgent.service = service
```

Comment:
is it a valid change? I don't see any updates in the tests to cover it

Reply:
Yes, this relate to a change made earlier for RUM to automatically use the channel's name as the service (like we do with the package name on android). On line 21 we have : 
```
service = configuration.service ?? channelServiceName()
```

Regarding the testing, unfortunately this part is not really testable with the way Roku works